lang/scala-docs/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                scala-docs
+categories          lang java
+license             Apache-2
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         The Scala Programming Language API docs
+long_description    {*}${description}
+
+homepage             https://www.scala-lang.org/
+platforms            darwin
```

Comment:
No need for `platforms darwin`, as this is now the default.

Reply:
Fixed